src/main/resources/application.yml

```
@@ -0,0 +1,66 @@
+          auth: true
+          ssl:
+            enable: true
+          starttls:
+            enable: true
+            required: true
+          connectionTimeout: 5000
+          timeout: 5000
+          writeTimeout: 5000
+        debug: true
+
+---
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)